lib/src/test/java/com/cloudbees/groovy/cps/CpsTransformerTest.java

```diff
@@ -965,30 +965,30 @@ public void spreadMapExpression() throws Throwable {
 
     @Test
     public void initialExpressionsInMethodsAreCpsTransformed() throws Throwable {
-        assertEquals(Boolean.FALSE, evalCPS(
+        assertEvaluate(Boolean.FALSE,
```

Comment:
I should have made these changes as part of https://github.com/cloudbees/groovy-cps/commit/9a3b1b042211e83b48fea6a15096e2575b397a0b, but I did not notice it until I was resolving the merge conflict with the newly added test at the bottom of the file.